### ISSUE TITLE
no password reset email

### ISSUE THREAD

**Issue by eroux** [buda-base/lds-pdi#196]
Karma tried to reset his password but didn't receive any email from auth0... not sure how to debug that but let's track it there

**Comment by xristy** [comment]
I just tried nd it didn't work for me either. Spam folders checked and such. 

An auth0 config somewhere?

**Comment by eroux** [comment]
hmm, how annoying... maybe... maybe we should contact support?

**Comment by MarcAgate** [comment]
this is here https://manage.auth0.com/dashboard/us/bdrc-io/templates/provider  . 

We are using our own email provider. Is someone able to test these settings? I tried to send a test email, Did someone receive it ?

**Comment by xristy** [comment]
It looks like contacting support after some reading will be needed. I tried a test email from the auth0 dashboard but never got it. Perhaps @MarcAgate has looked the auth0 config info? I don't see where our customized login and reset page templates are.

**Comment by eroux** [comment]
I think it's most likely a smtp configuration issue... cf. my private message on Skype

**Comment by MarcAgate** [comment]
@xristy : I gave above the link where you can view the smtp settings for our declared custom provider.
For the template, it's here: https://manage.auth0.com/dashboard/us/bdrc-io/templates

I tried to send a test email for the reset template and didn't get any email either.

We could eventually disable "Use my own email provider" to see how it goes and make sure the issue is coming from our provider settings.

**Comment by xristy** [comment]
@MarcAgate yes I have been visiting those links. The problem I have is that I have yet to see one of these emails

I don't know what email provider is used if we disable "Use my own email provider". The creds that are there allow me to login to mail.google.com as auth0.admin@tbrc.org, but I suspect that we need to tell google that mail originating from the _less secure_ auth0 mail client is legitimate

**Comment by xristy** [comment]
@MarcAgate my earlier comment about templates is in reference to [Universal Login](https://manage.auth0.com/dashboard/us/bdrc-io/login_page) where neither the **login** or the **password reset** templates look like what is currently presented at library.bdrc.io. So I wondered where that is being managed

**Comment by MarcAgate** [comment]
right there: https://manage.auth0.com/dashboard/us/bdrc-io/password_reset

There must be a google account associated with these settings: which one is it ? That's where these settings must probably be changed. 
We could eventually get rid of google and use something else (what do we use for tbrc.org ?).

**Comment by xristy** [comment]
Perhaps @berger-n knows? We use google for tbrc.org. autyh0.admin@tbrc.org is the only account I know of.

**Comment by eroux** [comment]
until January the gmail thing was sending recovery emails (you can see them in the sent folder in Gmail), and then it stops, probably because it was flagged as insecure by Google, now that it's not anymore let's see if emails work again

**Comment by eroux** [comment]
the auth0 test emails are sent to the auth0 tenant primary address which is support@tbrc.org. @xristy do you have access to this mailbox?

**Comment by xristy** [comment]
no I don't have the pw anymore. @jimk-bdrc changed it a while ago I think

**Comment by xristy** [comment]
@eroux 
> now that it's not anymore let's see if emails work again

are you saying that auth0 sending via auth0.admin@tbrc.org is no longer considered insecure?

If so how are seeing that?

Otherwise I think we'll have to setup an application password for auth0 to use.

**Comment by xristy** [comment]
@eroux I think we need to set up an app pw for auth0.admin@tbrc.org that we use in the auth0 _Emails_ / _Provider_. I think I did that when initially setting stuff up and that now the app pw has expired.

That's what appeared to happen to my person emal from my Mac Mail app a couple of weeks ago


No warning at all.

**Comment by xristy** [comment]
I think we probably ought to change the Tenant support email to something like  info@bdrc.io   to handle user's having issues with auth0

**Comment by xristy** [comment]
We have set up 2FA on auth0.admin@tbrc.org w/ my phone number for now. And then generated an app pw for auth0.com and plugged that into the provider info on auth0.com for the bdrc.io tenant. 

Password reset now seems to work although the message got stuck in my gmail spam folder.

We'll explore a 2FA approach using multiple recovery email addresses rather than depending on a single phone number.
